### ISSUE TITLE
Made diff about *.pbf ignore features order

### ISSUE THREAD

**Issue by sharkAndshark** [urbica/martin#1630]
[CI failed](https://github.com/maplibre/martin/actions/runs/12463285581/job/34785923272) due to the features order changed.

|feature|test on latest(ubuntu 24.04 + postgis/postgis:16-3.5)|other pg version in CI|
|----|----|----|
|cmp_0_0_0.pbf.txt, feature of layer 1 which its gid == 19|29th feature|18th feature|

A  `vt-show` which don't care features order might be needed.

**Comment by sharkAndshark** [comment]
PostGIS couldn't guarantee the features order without `order by`, so our ci failed on latest PostGIS, `postgis/postgis:16-3.5`.

Maybe we need a `diff` which ignore the features order.  @nyurik

**Comment by sharkAndshark** [comment]
To duplicate this failure just update the `docker-compose.yml` and run `just bless` or `just test`

```yml
........
  db-is-ready:
    # This should match the version of postgres used in the CI workflow
    image: postgis/postgis:16-3.5
........
  db:
    # This should match the version of postgres used in the CI workflow
    image: postgis/postgis:16-3.5
........
```

**Comment by sharkAndshark** [comment]
@nyurik we should fix this how about make a new vt-show tool? CC @CommanderStorm

**Comment by nyurik** [comment]
yeah, perhaps a simple tile content dumper, possibly using geozero for the conversion into geojson or csv, and sorting it?  I don't think Martin should contain the needed code to decode protobuf itself.

**Comment by sharkAndshark** [comment]
Agree. It should be a repo outside which read `*.pbf`, sort the features before writing. CSV or GeoJSON To Be Decided.

## How to sort?
Join all the properties key value paris, geometry type and coordiantes and then make a hash?

**Comment by nyurik** [comment]
I don't think we need a separate repo - this is just a few lines of code we would need, and maybe no code at all - just install the geozero cli tool.  See https://github.com/georust/geozero  - and if the list of features is exported to CSV, you can run it through `... | sort` in a linux shell.  Or if you want to store geojson, perhaps use `jq` to sort a specific array (this one is a bit trickier as you would need to use jq script)

**Comment by sharkAndshark** [comment]
Not be aware that it's a CLI tool either. I thought it's a lib only.. lollll Yeah. It's better make this in shell script.

**Comment by nyurik** [comment]
yeah, it is not auto-published (i just created a ticket for that). I guess we can simply do a few line binary if we can't get cli to do what we need, perhaps as a unpublished test tool.

**Comment by sharkAndshark** [comment]
So let's try shell + csv first. Seems it's the best solution.

**Comment by CommanderStorm** [comment]
> perhaps use jq to sort a specific array (this one is a bit trickier as you would need to use jq script)

For reference: we already have merged something similar ^^
https://github.com/maplibre/martin/blob/26249e217eb1d802a11e511bbaab92372ef91d69/tests/test.sh#L86C18-L86C32
